pkg/cmd/roachtest/run.go

```diff
@@ -118,6 +132,32 @@ func runTests(register func(registry.Registry), filter *registry.TestFilter) err
 		opt.debugMode = NoDebug
 	}
 
+	if roachtestflags.SideEyeApiToken != "" {
+		if roachtestflags.Local {
```

Comment:
Nit: could move the body to a helper func. for readability; `runTests` is already plentiful with boilerplate.